src/Service/Health/DiskSpaceHealthIndicator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akondas\ActuatorBundle\Service\Health;
+
+class DiskSpaceHealthIndicator implements HealthIndicator
+{
+    private string $cacheDir;
+    private int $threshold;
+
+    public function __construct(string $cacheDir, int $threshold)
```

Comment:
why cache dir?

Reply:
I initially thought that you want to monitor the internal cache directory, but it would lead to unexpected behaviour when you configure it outside your application directory

I've renamed this variable